src/Raven.Server/Documents/Revisions/RevisionsStorage.ManuallyDelete.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Raven.Server.Documents.TransactionMerger.Commands;
+using Raven.Server.ServerWide.Context;
+using Voron;
+using Voron.Data.Tables;
+
+namespace Raven.Server.Documents.Revisions;
+
+public partial class RevisionsStorage
+{
+    public bool IsAllowedToDeleteRevisionsManually(string collection)
+    {
+        var configuration = GetRevisionsConfiguration(collection);
+        if (configuration == ConflictConfiguration.Default)
+            return false;
+
+        return configuration.AllowDeleteRevisionsManually;
+    }
+
+    public Task DeleteRevisionsByChangeVectorManuallyAsync(List<string> cvs, long maxDeletes, bool shouldThrowIfChangeVectorsNotFound = true)
+    {
+        if (cvs == null || cvs.Count == 0)
+            return Task.CompletedTask;
+
+        if (cvs.Count > maxDeletes)
```

Comment:
I don't think that this is a problem. Both of those values came directly from the user, so we can process that.
`MaxDeletes` refers to deletion 1M revisions for `users/1` - for example. Without explicitly specifying this

Reply:
Done